### ISSUE TITLE
Can't resolve '../../shared/component'

### ISSUE THREAD

**Issue by adriantoine** [FormidableLabs/image-palette#25]
Hi!

I have just tried the React component but I always get this error:

```
./node_modules/react-image-palette/es/provider.js
Module not found: Can't resolve '../../shared/component' in '/Users/adrien/Projects/image-palette/demo/node_modules/react-image-palette/es'
```

The issue is quite easy to reproduce, check out this repo and run:
```
cd demo
yarn
yarn add react-image-palette@latest
yarn start
```
and you will see the error in the `create-react-app` output. The version used in the demo (`0.1.3`) doesn't have this issue, it only appears after upgrading.

**Comment by Godefroy** [comment]
Same problem on my project with parcel.

**Comment by aweary** [comment]
Sorry about that, looks like I just need to republish from master. I'll put out a patch release soon.

**Comment by adriantoine** [comment]
No problem, thanks!

**Comment by aweary** [comment]
I've published `0.2.4`, this should be fixed now!
